minard/views.py

```diff
@@ -348,12 +348,26 @@ def trigger():
 @app.route('/nearline')
 @app.route('/nearline/<int:run>')
 def nearline(run=None):
+    warning = []
     if run is None:
-	run = int(redis.get('nearline:current_run'))
+        run = int(redis.get('nearline:current_run'))
+        detector_run = get_current_run()
+        if run != detector_run - 1:
+            warning.append(run)
+            warning.append(detector_run)
 
     programs = redis.hgetall('nearline:%i' % run)
 
-    return render_template('nearline.html', run=run, programs=programs)
+    # Get failures over last 100 runs
```

Comment:
It would be nice to turn this into an argument you could pass like snopl.us/monitoring/nearline?limit=100 and then just have it default to 100.